docs/troubleshooting/troubleshoot_deployment.md

```diff
@@ -59,9 +63,17 @@ After removing the entry from the Terraform state, `terraform apply` should work
 
 You may see one or both of these error the first time you run `terraform apply` for your Cumulus deployment.
 
-**Cause:** The Amazon service for managing roles and permissions, IAM, [is eventually consistent](https://docs.aws.amazon.com/IAM/latest/UserGuide/troubleshoot_general.html#troubleshoot_general_eventual-consistency), meaning there can be a delay between when permission updates are applied and when they actually take effect. As a result, when doing a Terraform deployment the role permissions required for a resource may not have fully taken effect before Terraform attempts to create that resource, so the deployment fails.
+::::note IAM issue
+
+The Amazon service for managing roles and permissions, **IAM**, [is eventually consistent](https://docs.aws.amazon.com/IAM/latest/UserGuide/troubleshoot_general.html#troubleshoot_general_eventual-consistency), meaning there can be a delay between when permission updates are applied and when they actually take effect. As a result, when doing a Terraform deployment the role permissions required for a resource may not have fully taken effect before Terraform attempts to create that resource, so the deployment fails.
```

Comment:
```suggestion
The Amazon service for managing roles and permissions, **IAM**, [is eventually consistent](https://docs.aws.amazon.com/IAM/latest/UserGuide/troubleshoot_general.html#troubleshoot_general_eventual-consistency), meaning there can be a delay between when permission updates are applied and when they actually take effect. As a result, when doing a Terraform deployment, the role permissions required for a resource may not have fully taken effect before Terraform attempts to create that resource so the deployment fails.
```
commas 😵‍💫

Reply:
@botanical  Actually, the new comma should be there. Thanks. However, the "so the deployment fails" would keep the comma (per Grammarly).

Reply:
This is why I'm a writer and not an editor. 😵‍💫